bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/README.md

```diff
@@ -0,0 +1,112 @@
+# NibeGW Hardware and Compiling hints
+
+
+## RS-485 Modules
+
+For setttings up a NibeGW you need a RS485 module. Most cheap modules out there are compatible with 5V voltage and therefore compatible with Arduino based hardware. Mostly you will get one of two commonly used designs:
+
+* Modules based on Max1348 chip, which has 2 pins on the Arduino side (RXD, TXD) + VCC + GND
+* Modules based on Max485 chip, which has 4 pins on the Arduino side (RO, RE, DE, DI) + VCC + GND
+
+Both types of modules work fine with NibeGW. The difference between the two is, that the Max485 chip needs to be switched between RX mode and TX mode manually while the Max1348 chip do this automatically. That is why you need an extra "direction pin" on the Arduino to switch the module with Max485 chip between the two modes.
+
+#### Wiring diagram for Max1348 based modules:
+
+```
+ TX  RX   5V  GND    Arduino
+ |   |    |    |
+ |   |    |    |
+ |   |    |    |
+ RX  TX  VCC  GND    Max1348 based module
+```
+
+#### Wiring diagram for Max485 (PIN2 is used as direction pin here):
+
+```
+ TX  RX  PIN2      5V  GND    Arduino
+ |   |    |        |    |
+ |   |    |        |    |
+ |   |    |---|    |    |
+ DI  RO   DE  RE  VCC  GND    Max485 bases module
+```
+
+
+## Ethernet Shield W5100
```

Comment:
ProDiNo boards have also Ethernet build in, so separated shield is not needed

Reply:
Good point - added.